### PR TITLE
[KF-1579] Self hosted runnners

### PR DESCRIPTION
**Pull request by beliaev-maksim**
run 1.7/stable test on self hosted runners

### CODE REVIEW

File: tests-bundle/1.7/test_1dot7.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    if rc != 0:
+        raise Exception(f"Dispatch failed with code: {rc}, \nstdout: {stdout}, \nstderr {stderr}")
+
+    # now wait for all apps
+    await ops_test.model.wait_for_idle(
+        status="active",
+        raise_on_blocked=False,
+        raise_on_error=True,
+        timeout=from_minutes(minutes=30),
+    )
```
Comment (i-chvets) [inline tests-bundle/1.7/test_1dot7.py:103]: Do we want to fetch public-url to confirm all is up?

Comment (beliaev-maksim) [reply tests-bundle/1.7/test_1dot7.py:103]: yes, I wanna run selenium as a next step

**Review by i-chvets — APPROVED** [review]
Approved.

File: tests-bundle/1.7/test_1dot7.py (hunk 1 above)
Comment (i-chvets) [reply tests-bundle/1.7/test_1dot7.py:103]: +1

**Pull request merged**